oriedita/src/main/java/oriedita/editor/drawing/tools/DrawingUtil.java

```diff
@@ -456,4 +477,23 @@ public static void drawCpLine(Graphics g, LineSegment s, Camera camera, LineStyl
             }
         }
     }
+
+    public static int cohenSutherlandRegion(int clipX, int clipY, Point point) {
+        return cohenSutherlandRegion(0,0,clipX, clipY, point);
+    }
+
+    public static int cohenSutherlandRegion(int clipLowX, int clipLowY, int clipHighX, int clipHighY, Point point) {
```

Comment:
Maybe add a reference or explanation here about the Cohen Sutherland Line Clipping Algorithm

Reply:
So that you know where to find more info when something needs to change here at some point.

Reply:
Alright, will do